python/tests/test_transforms.py

```diff
@@ -129,6 +131,52 @@ def test_string_with_surrogate_pair():
     assert bucket_transform.hash(string_with_surrogate_pair) == mmh3.hash(as_bytes)
 
 
+@pytest.mark.parametrize(
+    "type_var,value,expected",
+    [
+        (LongType(), None, "null"),
+        (DateType(), 17501, "2017-12-01"),
+        (TimeType(), 36775038194, "10:12:55.038194"),
+        (TimestamptzType(), 1512151975038194, "2017-12-01T18:12:55.038194Z"),
```

Comment:
Timestamptz literals must always include a valid zone offset that matches this regex: `[+-]\d\d:\d\d$`

Reply:
👌 Fixed.